src/main/kotlin/exercise2/task1/FindPairOfHighestSum.kt

```diff
@@ -18,7 +18,17 @@ import org.jetbrains.exercise2.task3.findPairWithBiggestDifference
  */
 
 internal fun List<Int>.findHighestSumPair(): Pair<Int, Int> {
-    TODO("Implement me!!")
+    var par = Pair(this.min(), this.min())
```

Comment:
`this.min()` iterira kroz ceo niz kako bi našao najmanji broj. 
Koristiti ga dva puta nije najperformantnije rešenje.

Ukoliko radite iterativnim pristupom, inicijalna vrednost `par` objekta je mogla da bude `null`.
Trudite se da variable imenujete na Engleskom jeziku. :)